### PR TITLE
Load activity and search

### PR DESCRIPTION
**Pull request by robhillman97**
Adds:

- load activity by selecting tile on home page
- open activity in edit or view mode
- search activities by all, title or author

### CODE REVIEW

File: Assets/MirageXR/Tests/NewUI/SearchView.cs [hunk 1]
```diff
@@ -1,9 +1,121 @@
...
+    [SerializeField] private TMP_InputField _inputFieldSearch;
+    [SerializeField] private Button _allButton;
+    [SerializeField] private Button _titleButton;
+    [SerializeField] private Button _authorButton;
+
+    private List<SessionContainer> _content;
+    private readonly List<ActivityListItem_v2> _items = new List<ActivityListItem_v2>();
+
+    private ActivityListView_v2 activityListView;
+    private int searchType;
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/SearchView.cs]: replace with Enum type

File: Assets/MirageXR/Tests/NewUI/ActivityListItem_v2.cs [hunk 2]
```diff
@@ -98,10 +103,24 @@ private async void OnBtnMain()
         {
             _interactable = false;
             if (!_container.ExistsLocally) await DownloadActivityAsync();
-            else await PlayActivityAsync();
+            else showPopup();
             _interactable = true;
         }
 
+        private void showPopup()
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/ActivityListItem_v2.cs]:
```suggestion
        private void ShowPopup()
```

File: Assets/MirageXR/Tests/NewUI/SearchView.cs [hunk 3]
```diff
@@ -1,9 +1,121 @@
...
+
+        EventManager.OnActivityStarted += Close;
+
+        searchType = 0;
+
+        UpdateListView();
+    }
+
+
+    public void OnAllClick() {
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/SearchView.cs]:
```suggestion
    public void OnAllClick() 
    {
```

**Review by cawke — CHANGES_REQUESTED** [review]

**Comment by robhillman97** [comment]
Edited to address requested changes

**Review by cawke — APPROVED** [review]
ty

**Pull request merged**